11/ex11.py

```diff
@@ -0,0 +1,27 @@
+#prints question (argument)
+print "How old are you?" ,
+#input field for age
+age = raw_input ()
```

Comment:
This runs fine, but like I mentioned, no spaces between the function name and call — `raw_input()`.